pkpdapp/pkpdapp/models/biomarker.py

```diff
@@ -0,0 +1,25 @@
+#
+# This file is part of PKPDApp (https://github.com/pkpdapp-team/pkpdapp) which
+# is released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+
+from django.db import models
+from pkpdapp.models import BiomarkerType, Dataset
+
+
+class Biomarker(models.Model):
+    """
+    A single biomarker measurement value stored in a :model:`pkpdapp.Dataset`.
+    Each biomarker is assigned a type stored in :model:`pkpdapp.BiomarkerType`
```

Comment:
End with a full stop? Just thinking it is probably good for us to always write in correct grammar English so as to avoid ambiguity. What do you think?

Reply:
yup, sounds good. I've added the stop.